src/buildstream/plugins/sourcemirrors/default.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional, Dict, List, Any, TYPE_CHECKING
+
+from buildstream import SourceMirror, MappingNode, SequenceNode, SourceMirrorError
+
+
+class AliasesSourceMirror(SourceMirror):
+    BST_MIN_VERSION = "2.1"  # temporary
```

Comment:
Do you want to fix this up before merging or what's the plan? Is there an issue with simply creating a 2.2.0.dev0 tag now?

Reply:
That sounds sensible to me, we should probably branch 2.1 just in case in advance, and we can have as many `dev0`, `dev1` etc tags as we want in master until 2.2.

Reply:
We already have a [bst-2.1 branch](https://github.com/apache/buildstream/tree/bst-2.1). It's just a matter of making a tag.